generic/tests/netperf.py

```diff
@@ -223,10 +253,12 @@ def _pin_vm_threads(vm, node):
             password = params_tmp["password"]
             username = params_tmp["username"]
             env_setup(i, ip_dict[i], username, shell_port, password)
+    mtu = params.get("mtu", "1500")
+    mtu_set(mtu)
```

Comment:
It's better to add if to judge mtu before execute mtu_set, because there is no need to execute it when mtu is default value.

Reply:
Yes, it's better to judge it before run mtu_set, I have updated it, please help review.

Reply:
I have updated it, please help review.